eZ/Bundle/EzPublishCoreBundle/Converter/ContentParamConverter.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * File containing the ContentParamConverter class.
```

Comment:
Since recently we use `This file is part of the eZ Publish Kernel package`. (should be applied on other new files here as well)